### PR TITLE
Update other references of media count to 700 million

### PR DESCRIPTION
**Pull request by krysal**
## Description

While looking at the same problem that #1098 resolves I thought of extracting the default meta tags to its own file, so the Nuxt configuration is reduced by a some lines and also creates a constant for the page description :) Not the ideal solution still but the number of items is more centralized.

## Testing Instructions

1. `just frontend/run dev`
2. Visit http://localhost:8443/ and check on the source page the "700 million" description is still showed correctly.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/constants/meta.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/**
+ * The default metadata for the site. Can be extended and/or overwritten per page. And even in components!
+ * See the Nuxt.js docs for more info.
+ * {@link https://nuxtjs.org/guides/features/meta-tags-seo} Nuxt.js Docs
+ */
```
Comment (dhruvkb) [inline frontend/src/constants/meta.ts:5]: I think this doc comment was associated with the `meta` variable on L10.

**Review by dhruvkb — APPROVED** [review]
I like the idea of `NUMBER_OF_RECORDS` being a constant 👍. 

Making it modifiable via an env var would've been nice but it'll break snapshots when changed so I don't mind this solution.

File: frontend/src/constants/meta.ts (hunk 1 above)
Comment (krysal) [reply frontend/src/constants/meta.ts:8]: Right, good catch! Thank you.

**Review by zackkrida — APPROVED** [review]
LGTM. I made a follow-up issue to address the fact that these strings aren't localized: #1129

**Pull request merged**